src/modules/personenkontext/api/dbiam-personenkontext-filter.controller.ts

```diff
@@ -62,9 +62,10 @@ export class DbiamPersonenkontextFilterController {
         @Query() params: FindPersonenkontextSchulstrukturknotenBodyParams,
     ): Promise<FindSchulstrukturknotenResponse> {
         const anlage: PersonenkontextAnlage = this.personenkontextAnlageFactory.createNew();
+        const sskName: string = params.sskName || '';
```

Comment:
Use `??` instead to make it more clear (Won't make a difference in this case, but an empty string will also resolve to false when testing with `||`. For example, `"" || "test"` will resolve to `test`. The null-coalescing-operator is more idiomatic)

Reply:
Done.